### ISSUE TITLE
Boardwalk Server

### ISSUE THREAD

**Issue by tonibm19** [zhuowei/Boardwalk#163]
If you managed to port the client, server should be easier, and it shouldnt lag a lot.

**Comment by joho506** [comment]
As in running the server on an Android device? That would (also) be hideously slow.

**Comment by tonibm19** [comment]
I dont think so. If you run it in a chroot yes but running directly in android it should work not that bad.
